readwiki.py

```diff
@@ -23,19 +23,6 @@ def set_data(self, _revision, _change):
     def set_message(self, _message):
         self.message = _message
 
-    def is_spam(self):
-        """
-        various checks to see if the change is/might be spam
-        """
-
-        if not self.hasData:
```

Comment:
i have a strong feeling that this is where we check for spam bots or auto. celaning spam messages, without it this will be go through, please keep

Reply:
I've re-added the method, but renamed it to `should_be_ignored`. Is that fine with you?